crypto/ec/build.info

```diff
@@ -29,8 +29,9 @@ IF[{- !$disabled{asm} -}]
 
   $ECASM_armv4=ecp_nistz256.c ecp_nistz256-armv4.S
   $ECDEF_armv4=ECP_NISTZ256_ASM
-  $ECASM_aarch64=ecp_nistz256.c ecp_nistz256-armv8.S
-  $ECDEF_aarch64=ECP_NISTZ256_ASM
+  $ECASM_aarch64=ecp_nistz256.c ecp_nistz256-armv8.S ecp_sm2p256.c \
+                 ecp_sm2p256_table.c ecp_sm2p256-armv8.S
+  $ECDEF_aarch64=ECP_NISTZ256_ASM ECP_SM2P256_ASM
```

Comment:
This needs to be conditional on sm2 being enabled:

```suggestion
  $ECASM_aarch64=ecp_nistz256.c ecp_nistz256-armv8.S
  $ECDEF_aarch64=ECP_NISTZ256_ASM
  IF[{- !$disabled{'sm2'} -}]
    $ECASM_aarch64=$ECASM_aarch64ecp ecp_sm2p256.c \
                   ecp_sm2p256_table.c ecp_sm2p256-armv8.S
    $ECDEF_aarch64=$ECDEF_aarch64 ECP_SM2P256_ASM
  ENDIF
```

Reply:
fixed on https://github.com/openssl/openssl/pull/20754/commits/106d6f4b6a61d0e517eb1c6053e8222665baeb52